index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>mobile skeleton</title>
+  <link rel="stylesheet" href="/style/style.css">
+  <!-- Bootrap Icon -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+</head>
+<body>
+  <header>
+    <div class="logo">
+      <p>Welcome</p>
+    </div>
+    <nav>
+      <a href="">Portfolio</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+    </nav>
+    <div class="menu">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </header>
+  <main>
+    <section id="headline">
+      <div class="fonts intro">
+        <h2>
+          Hey There. I'm Josh
+        </h2>
+      </div>
+      <div class="fonts bg">
+        <h2>
+          I'm a Software Engineer
+        </h2>
+      </div>
+      <div class="more">
+        I can help you build a product, feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don't hesitate to contect me or 
+        <a href="https://ko-fi.com/https://ko-fi.com/joshbatey" class="coffee"> 
+          <img src="https://cdn.ko-fi.com/cdn/kofi3.png?v=3" height="30" width="110" alt="https://ko-fi.com/joshbatey" />
+        </a>
+      </div>
```

Comment:
This content should be wrapped in a `<p>` tag, please fix it.

<img width="339" alt="Screen Shot 2022-06-30 at 8 01 15" src="https://user-images.githubusercontent.com/82426682/176683563-68ea6094-926c-4fbf-a1e8-8a53774ff3b5.png">